### ISSUE TITLE
Just found out there is a CACHE list

### ISSUE THREAD

**Issue by SuperCuek** [web-scrobbler/web-scrobbler#5048]
can anyone help to clear all cache list in few clicks? thanks
![image](https://github.com/user-attachments/assets/721929e4-de53-48f3-a3bc-1a47b3af7461)

**Comment by inverse** [comment]
Yeah the extension caches scrobbles that failed to be submitted due to certain issues to be retried later on.

We don't have a clear all cache button but it could be something we could support.

**Comment by AFlowOfCode** [comment]
Here is how you clear the cache:
1. Go to the extension settings page
2. Open devtools & click on console tab
3. `>> browser.storage.local.remove('ScrobbleCache')`

If you want to see everything the extension stores, run this first: `>> browser.storage.local.get()`
You'll see a `Promise` object - open that, then open "value". It'll show you all the keys & their values. This works for any extension.
Here's an example just looking up what's stored for "ScrobbleCache".
![Image](https://github.com/user-attachments/assets/5e0adc68-3483-439f-bbd4-f2c294904c91)

However there should definitely be a simple, user-friendly option to clear the cache. There should also be clarifying details in the UI as to how it behaves. For example - is it infinite or automatically cleared at some point down the line?

I can see [here](https://github.com/web-scrobbler/web-scrobbler/blob/dc821798272ce50abeb9c491ca4e22aeaa4fdc8a/src/core/storage/scrobble-cache.model.ts#L10) that the max number of items is currently set to 1000. That should be in the UI as well.

Many people find this type of "surprise" data collection to be off-putting. 

For example, mine has a list of pretty much every video (music or not) I've watched on Youtube since I've installed the extension. This is concerning from a privacy perspective, especially considering 
* you aren't told about it up front
* there's no built-in option to clear it
* it tracks literally everything on every related site

So why save all this stuff? It's clearly not just items that failed to scrobble & need to be retried. [Here](https://github.com/web-scrobbler/web-scrobbler/blob/dc821798272ce50abeb9c491ca4e22aeaa4fdc8a/src/core/storage/wrapper.ts#L119) we see it stores pretty much everything it comes across, whether they were successful, disallowed (due to not being recognized as music in my settings, for example), ignored, etc:
```
export enum ScrobbleStatus {
	SUCCESSFUL = 'success',
	IGNORED = 'ignored',
	ERROR = 'error',
	DISALLOWED = 'disallowed',
	INVALID = 'invalid',
}
``` 

I don't need an extension tracking things that don't apply to it (disallowed/ignored/invalid) and things I'm already using an account to track (successful), which is the actual purpose of the extension in the first place. So this just seems like a pure privacy violation waiting to happen, or simply a big misstep.
 
According to the privacy policy, no data leaves my local device. If this were being sent anywhere I'd consider this extension to be spyware first and foremost & only providing its functionality as a way in the door. I didn't go through every part of the codebase & actual installed extension code to verify the storage isn't ever exfiltrated. I would hope based on good faith of the developers that it is not. 

Either way, it's 100% unnecessary to be storing anything except perhaps the "error" category for a certain number of retries before calling it quits & removing. Purely temporary and functional. 

Nothing else has any legitimate purpose (within the extension's realm of service) & probably just makes people suspicious about being tracked. All it's doing is creating an extra set of obscurely hidden personal data that a user probably isn't aware of or has no idea how to get rid of (besides uninstalling & not using the extension). 

Remember - anywhere there's data collected, there's a risk of an unwanted leak. This is a cool extension & I thank you for it, but this part doesn't fit!
